### PR TITLE
OrbitControls: Add `stopListenToKeyEvents()`.

### PR DESCRIPTION
**Pull request by nikolas**
If I called listenToKeyEvents(), sometimes I want to remove this event listener, temporarily or otherwise.

### CODE REVIEW

File: examples/jsm/controls/OrbitControls.js [hunk 1]
```diff
@@ -124,6 +124,13 @@ class OrbitControls extends EventDispatcher {
 
 		};
 
+		this.removeKeyEvents = function () {
+
+			domElement.removeEventListener( 'keydown', onKeyDown );
+			this._domElementKeyEvents = null;
```
Comment (Mugen87) [inline examples/jsm/controls/OrbitControls.js:130]: I think this should be:
```js
this._domElementKeyEvents.removeEventListener( 'keydown', onKeyDown );
this._domElementKeyEvents = null;
```
You are referring to `domElement` which is not the DOM element for key events.

Comment (nikolas) [reply examples/jsm/controls/OrbitControls.js:130]: You're right - thank you. I'll update this unless you get to it first.

Comment (Mugen87) [reply examples/jsm/controls/OrbitControls.js:130]: Done!

I just wonder if `removeKeyEvents()` is a good name (next to `listenToKeyEvents()`). 

@mrdoob What are your thoughts about this?

Comment (nikolas) [reply examples/jsm/controls/OrbitControls.js:130]: Yeah, my first thought was "unlistenToKeyEvents()" so that's another option.

Comment (Mugen87) [reply examples/jsm/controls/OrbitControls.js:130]: I like this name even more. I was at first unsure about the "unlisten" term, but after read [this](https://ell.stackexchange.com/questions/214085/is-there-any-word-for-unlisten-negative-of-listen), I'm convinced it's good 😊 .

**Review by Mugen87 — APPROVED** [review]

**Comment by WestLangley** [comment]
"unlisten" is not a word. Is there a better API? 🤔 

Also, maybe set `this._domElementKeyEvents = null;` on `.dispose()`.

**Comment by Mugen87** [comment]
> "unlisten" is not a word. Is there a better API?

https://ell.stackexchange.com/questions/214085/is-there-any-word-for-unlisten-negative-of-listen

**Comment by Mugen87** [comment]
> Also, maybe set this._domElementKeyEvents = null; on .dispose().

Good catch! I'll update the PR.

**Comment by WestLangley** [comment]
>In the context of programming, it's better to be clear than to try and force "correct" English.
>>TBH, I feel very strong about this statement.

That's fine. But if you are using "unlisten" to mean "stop listening", then why not use `stopListeningToKeyEvents`.

That's both correct and clear.

Or `stopListenKeyEvents`.

**Comment by WestLangley** [comment]
>In the context of programming, it's better to be clear than to try and force "correct" English.
>>TBH, I feel very strong about this statement.

That's fine. But if you are using "unlisten" to mean "stop listening", then not use `stopListeningToKeyEvents`.

That's both correct and clear.

Or `stopListenKeyEvents`.

**Comment by WestLangley** [comment]
>In the context of programming, it's better to be clear than to try and force "correct" English.
>>TBH, I feel very strong about this statement.

That's fine. But if you are using "unlisten" to mean "stop listening", then use `stopListeningToKeyEvents`.

That's both correct and clear.

Or `stopListenKeyEvents`.

**Comment by Mugen87** [comment]
How about `stopListenToKeyEvents()`? I think it's better for programming when the `*ListenToKeyEvents()` suffix is the same for both methods.

**Comment by WestLangley** [comment]
Oh yes. (That was a typo.)

**Comment by mrdoob** [comment]
> "unlisten" is not a word. Is there a better API? 🤔

Can we make it a word? 😅

How about `ignoreKeyEvents()`?

```js
listenKeyEvents()
ignoreKeyEvents()
```

I don't like that "To" in the name... 😕

**Comment by Mugen87** [comment]
The `*to*` names were discussed here: https://github.com/mrdoob/three.js/pull/21054#discussion_r555254757.

I'm okay with the shorter version as well :)

**Pull request merged**